libraries/botbuilder-ai/botbuilder/ai/qna/models/query_results.py

```diff
@@ -9,11 +9,22 @@
 class QueryResults:
     """ Contains answers for a user query. """
 
-    def __init__(self, answers: List[QueryResult]):
+    _attribute_map = {
+        "answers": {"key": "answers", "type": "[QueryResult]"},
+        "active_learning_enabled": {"key": "activeLearningEnabled", "type": "bool"},
+    }
+
+    def __init__(
+        self, answers: List[QueryResult], active_learning_enabled: bool = None, **kwargs
+    ):
         """
         Parameters:
         -----------
 
         answers: The answers for a user query.
+
+        active_learning_enabled: The active learning enable flag.
         """
+        super(QueryResults, self).__init__(**kwargs)
```

Comment:
The same is true for the **kwargs argument and the super() initialization call, if you don't inherit from msrest model they are not necessary.